.github/workflows/pull_request_automation.yml

```diff
@@ -31,4 +31,20 @@ jobs:
 
     - name: isort
       run: |
-        isort . --profile black --filter-files --check-only --diff --skip venv
\ No newline at end of file
+        isort . --profile black --filter-files --check-only --diff --skip venv
+        
+    - name: Before Build
```

Comment:
please see how CI is actually done in IntelOwl. Please split linters with the tests

Reply:
Done